djangocms_navigation/models.py

```diff
@@ -0,0 +1,58 @@
+from django.conf import settings
+from django.contrib.contenttypes.fields import GenericForeignKey
+from django.contrib.contenttypes.models import ContentType
+from django.contrib.sites.models import Site
+from django.db import models
+from django.utils.translation import ugettext_lazy as _
+from treebeard.mp_tree import MP_Node
+
+
+class Menu(models.Model):
+    """
+    MenuContent Grouper
+    """
+    pass
+
+
+class MenuContent(models.Model):
+    title = models.CharField(verbose_name=_('title'), max_length=100)
+    site = models.ForeignKey(
+        Site,
+        on_delete=models.PROTECT,
+    )
+    menu = models.OneToOneField(
+        Menu,
+        on_delete=models.CASCADE,
+        primary_key=True,
+    )
+    modified_by = models.ForeignKey(
+        settings.AUTH_USER_MODEL,
+        on_delete=models.PROTECT,
+        verbose_name=_('modified_by'),
+        null=True,
```

Comment:
Should this ever be null? If we're doing PROTECT on delete, then I'm not sure there's a reason to have null=True actually.